src/highlighting.c

```diff
@@ -1705,6 +1707,11 @@ gboolean highlighting_is_string_style(gint lexer, gint style)
 				style == SCE_DART_RAWSTRING_DQ ||
 				style == SCE_DART_TRIPLE_RAWSTRING_SQ ||
 				style == SCE_DART_TRIPLE_RAWSTRING_DQ);
+
+		case SCLEX_NIX:
+			return (style == SCE_NIX_STRING ||
+				style == SCE_NIX_STRING_MULTILINE ||
+				style == SCE_NIX_ESCAPECHAR);
```

Comment:
Should `SCE_NIX_OPERATOR_STRING` and/or `SCE_NIX_KEY` be added here? I have no idea what those are but they sound like they might be relevant.

Reply:
I don't think so:
- `SCE_NIX_OPERATOR_STRING` is style for `$()` in string interpolation
- `SCE_NIX_KEY` is `key` in `key= "hello";`